src/modules/45-dashboards/pages/dashboardView/DashboardView.tsx

```diff
@@ -81,7 +85,10 @@ const DashboardViewPage: React.FC = () => {
   React.useEffect(() => {
     const generateSignedUrl = async (): Promise<void> => {
       const { resource } = (await createSignedUrl()) || {}
-      setEmbedUrl(resource)
+      if (resource && resource !== '') {
```

Comment:
The empty string check is a bit superfluous as an empty string is already falsy
```suggestion
      if (resource) {
```